lab_2_retrieval_w_bm25/main.py

```diff
@@ -42,11 +42,17 @@ def remove_stopwords(tokens: list[str], stopwords: list[str]) -> list[str] | Non
 
     In case of corrupt input arguments, None is returned.
     """
+<<<<<<< HEAD
```

Comment:
what is it?

Reply:
i don't know how it is happened, but I fixed it